### PR TITLE
antibody, apg: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/antibody.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# antibody
+
+> Le manager de module shell le plus rapide.
+> Plus d'informations : <https://getantibody.github.io>.
+
+- Regroupe tous modules pour un chargement statique :
```
Comment (nicokosi) [inline pages.fr/common/antibody.md:6]:
```suggestion
- Regroupe tous les modules pour un chargement statique :
```

File: pages.fr/common/antibody.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`antibody bundle < {{~/.zsh_modules.txt}} > {{~/.zsh_modules.sh}}`
+
+- Mets à jour tous les modules :
```
Comment (nicokosi) [inline pages.fr/common/antibody.md:10]:
```suggestion
- Met à jour tous les modules :
```

Comment (Nico385412) [reply pages.fr/common/antibody.md:10]: Hi ! 

I must have an `s` at the end 😅 
image

File: pages.fr/common/antibody.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# antibody
+
+> Le manager de module shell le plus rapide.
```
[the remark this answers is not in the document]
Comment (Nico385412) [reply pages.fr/common/antibody.md:3]: You're definitely right ! 

I'll update it :D (both french and english page)

[the remark this answers is not in the document]
Comment (Nico385412) [reply pages.fr/common/antibody.md]: done :D

File: pages/common/antibody.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # antibody
 
-> The fastest shell plugin manager.
+> "The fastest" shell plugin manager.
```
Comment (EmilySeville7cfg) [inline pages/common/antibody.md:3]: Why quotes are added?

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages.fr/common/antibody.md (hunk 2 above)
Comment (nicokosi) [reply pages.fr/common/antibody.md:10]: OK, I think both are correct : `mets` for "(you) update" (_impératif présent_) and `met` is for "it updates" (_indicatif présent_). That's why I suggested `met`. :nerd_face: :sweat_smile: 

See https://www.larousse.fr/conjugaison/francais/mettre/6126 : 

> INDICATIF
> -Présent
> je mets
> tu mets
> il, elle met
> 
> IMPÉRATIF
> -Présent
> mets
> mettons
> mettez

File: pages.fr/common/antibody.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# antibody
+
+> Le manager de module shell "le plus rapide".
```
Comment (nicokosi) [inline pages.fr/common/antibody.md:3]:
```suggestion
> Le gestionnaire de module shell "le plus rapide".
```

**Review by nicokosi — APPROVED** [review]
Looks good to me! :+1:

File: pages.fr/common/antibody.md (hunk 2 above)
Comment (Nico385412) [reply pages.fr/common/antibody.md:10]: ooh that why !

But in the style-guide, they recommand imperative :p 

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood

File: pages.fr/common/antibody.md (hunk 5 above)
Comment (Nico385412) [reply pages.fr/common/antibody.md]: nice catch, i'll be more careful next time with `manager` :p

File: pages.fr/common/antibody.md (hunk 2 above)
Comment (marchersimon) [reply pages.fr/common/antibody.md:10]: Well, imperative makes the most sense in English (and a lot of other language). But we should use what feels the most natural in a specific language.

Comment (Nico385412) [reply pages.fr/common/antibody.md:10]: For me i rather like imperative than indicative. IMHO 😅

File: pages/common/antibody.md (hunk 4 above)
Comment (navarroaxel) [inline pages/common/antibody.md:3]: What about a command description without a subjective descriptions?
```suggestion
> Shell plugin manager with focus in performance.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/antibody.md (hunk 4 above)
Comment (marchersimon) [reply pages/common/antibody.md:3]: @navarroaxel I think it should be `focus on performance`. Also, this PR is only for translations, should we really add this change here?

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**